lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -544,12 +546,30 @@ def set_3d_properties(self, zs, zdir):
             xs = []
             ys = []
         self._offsets3d = juggle_axes(xs, ys, np.atleast_1d(zs), zdir)
-        self._facecolor3d = self.get_facecolor()
-        self._edgecolor3d = self.get_edgecolor()
-        self._sizes3d = self.get_sizes()
-        self._linewidth3d = self.get_linewidth()
+        # grab the current sizes as-is
+        # In the base draw methods we access the attributes directly which
+        # means we can not buffer the shuffling on the way out.
```

Comment:
I don't quite understand this comment; what does buffer mean here?

Reply:
Fair, I'll re-word that.